doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" importdescriptors "$(cat ~/descriptors.json)"
+[
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  }
+]
+```
+
+### Send Funds to the Offline Wallet
+
+1. Generate a new address in the `offline_wallet` to receive funds.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" getnewaddress
+tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv
+```
+
+2. Send funds from the `online_sending_wallet` to the generated `offline_wallet` address.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_sending_wallet" sendtoaddress "tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv" 0.0009
+0408d91070a7d02f714066b4b1464f2e0040dce467f8a9edf24255a871a2e865
+```
+
+3. Confirm the received funds (to the `offline_wallet`) using the online `watch_only_wallet`.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" listunspent
+[
+  {
+    "txid": "0408d91070a7d02f714066b4b1464f2e0040dce467f8a9edf24255a871a2e865",
+    "vout": 1,
+    "address": "tb1q6xp92v5jfllewp9gum9mgqlgsfvefrzrvrc8tv",
+    "label": "",
+    "scriptPubKey": "0014d1825532924fff9704a8e6cbb403e88259948c43",
+    "amount": 0.00090000,
+    "confirmations": 1,
+    "spendable": true,
+    "solvable": true,
+    "desc": "wpkh([413037c1/84h/1h/0h/0/0]02b66303a69d660e98c49320afc28dd9abd8f34c9d59c4ca2148a243fa811ea6c8)#6d9cgsep",
+    "parent_descs": [
+      "wpkh([413037c1/84h/1h/0h]tpubDCjzsbgeDYJj2Xgpb5T5CjQ97muocfoQDkYu68RJvu4sspunz82G7pBfFcWUvCGj4AhQoDkbX24jxXyrREyVUAba6B8qKGkFNPyA8GBQ3kA/0/*)#3us38eqx"
+    ],
+    "safe": true
+  }
+]
+```
+
+## Step 2: Sending money from the offline wallet
+
+### Create an Online Receiving Wallet
+
+1. On the online machine, create an online wallet named `online_receiving_wallet`.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="online_receiving_wallet"
+{
+  "name": "online_receiving_wallet"
+}
+```
+
+2. Generate a new address on the `online_receiving_wallet` to receive funds from the offline wallet.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_receiving_wallet" getnewaddress
+tb1q9k5w0nhnhyeh78snpxh0t5t7c3lxdeg3erez32
+```
+
+### Create and Export a PSBT Transaction
+
+1. Create a PSBT transaction using the online `watch_only_wallet`, sending funds to the `online_receiving_wallet` address.
+- Export the psbt transaction to `funded_psbt.txt` for easy portability to the `offline_wallet`
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" walletcreatefundedpsbt '[]' '[{"tb1q9k5w0nhnhyeh78snpxh0t5t7c3lxdeg3erez32": 0.0008}]' | jq -r '.psbt' >> ~/funded_psbt.txt
```

Comment:
Might be nice to explain the arguments here (inputs[], outputs[]). And again explain what you're doing with `jq`